data/SrimanNarayana7.json

```diff
@@ -0,0 +1,19 @@
+{
+"name": "SrimanNarayana",
+"displayStatsPublic": true,
+"type": "personal",
+"bio": "Software Developer and Tech-Enthusiast, learning Blockchain Technology and Java Full stack",
+"avatar": "https://github.com/SrimanNarayana7.png"
```

Comment:
We no longer need the `avatar` key, it will be automatically picked from your GitHub profile. So also remove this field from your json. 

[Updated docs](https://linkfree.eddiehub.io/docs/quickstart)
```suggestion
```